### PR TITLE
MIFOSX-1958: Payment details is not getting displayed if any transaction...

### PR DESCRIPTION
**Pull request by rajeshreddy-k**
... detail is viewed from Search Journal entry screen

### CODE REVIEW

File: app/global-translations/locale-en.json [hunk 1]
```diff
@@ -786,6 +786,7 @@
     "label.heading.accountusage": "Account Usage",
     "label.heading.viewaccounttransferdetails": "View Account Transfer Details",
     "label.heading.transactiondetails": "Transaction Details",
+    "label.heading.paymentdetails": "Payment Details:",
```
Comment (vishwasbabu) [inline app/global-translations/locale-en.json]: this label key already exists

File: app/global-translations/locale-en.json [hunk 2]
```diff
@@ -857,6 +858,10 @@
     "label.input.accountingrules": "Accounting rules",
     "label.input.credit": "Credit",
     "label.input.debit": "Debit",
+    "label.input.paymenttype": "Payment Type",
+    "label.input.accountnumber": "Account Number",
+    "label.input.chequenumber": "Cheque#",
+    "label.input.bankname": "Bank#",
```
Comment (vishwasbabu) [inline app/global-translations/locale-en.json]: labels like

"label.input.checknumber": "Cheque#",
    "label.input.routingcode": "Routing code",
    "label.input.receiptnumber": "Receipt#",
    "label.input.banknumber": "Bank#",

already exist

**Comment by vishwasbabu** [comment]
@rajeshreddy-k The pr looks good :)

Can you please resend the pr after removing the new labels being added (similar labels already exist, please use the same)

File: app/global-translations/locale-en.json (hunk 1 above)
Comment (rajeshreddy-k) [inline app/global-translations/locale-en.json]: yes sit that label key is already present sir, sorry sir i dint check. can
i modify and send pull request again sir.

On Mon, Apr 13, 2015 at 1:32 PM, Vishwas Babu A J <notifications@github.com>
wrote:

> In app/global-translations/locale-en.json
> <https://github.com/openMF/community-app/pull/1327#discussion_r28220258>:
>
> > @@ -786,6 +786,7 @@
> >      "label.heading.accountusage": "Account Usage",
> >      "label.heading.viewaccounttransferdetails": "View Account Transfer Details",
> >      "label.heading.transactiondetails": "Transaction Details",
> > +    "label.heading.paymentdetails": "Payment Details:",
>
> this label key already exists
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openMF/community-app/pull/1327/files#r28220258>.
>



-- 
Thank you & regards
Rajesh Reddy k
7795405419

Comment (rajeshreddy-k) [inline app/global-translations/locale-en.json]: yes sir* that label key is already present sir, sorry sir i dint check. can
i modify and send pull request again sir.

On Mon, Apr 13, 2015 at 1:40 PM, Rajesh Reddy <
rajesh.reddy@confluxtechnologies.com> wrote:

> yes sit that label key is already present sir, sorry sir i dint check. can
> i modify and send pull request again sir.
>
> On Mon, Apr 13, 2015 at 1:32 PM, Vishwas Babu A J <
> notifications@github.com> wrote:
>
>> In app/global-translations/locale-en.json
>> <https://github.com/openMF/community-app/pull/1327#discussion_r28220258>:
>>
>> > @@ -786,6 +786,7 @@
>> >      "label.heading.accountusage": "Account Usage",
>> >      "label.heading.viewaccounttransferdetails": "View Account Transfer Details",
>> >      "label.heading.transactiondetails": "Transaction Details",
>> > +    "label.heading.paymentdetails": "Payment Details:",
>>
>> this label key already exists
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/openMF/community-app/pull/1327/files#r28220258>.
>>
>
>
>
> --
> Thank you & regards
> Rajesh Reddy k
> 7795405419
>



-- 
Thank you & regards
Rajesh Reddy k
7795405419

**Pull request closed without merging**